packages/monitors/src/SyntheticPegMonitor.js

```diff
@@ -272,12 +290,14 @@ class SyntheticPegMonitor {
   // Takes in two big numbers and returns the error between them. using: δ = (observed - expected) / expected
   // For example an observed price of 1.2 with an expected price of 1.0 will return (1.2 - 1.0) / 1.0 = 0.20
   // This is equivalent of a 20 percent deviation between the numbers.
-  // Note that this logger can return negative error if the deviation is in a negative direction.
+  // Note 1) this logger can return negative error if the deviation is in a negative direction. 2) Regarding scaling,
+  // prices can be scaled arbitrarily but this function always returns 1e18 scaled number as a deviation error is
+  // a unitless number.
   _calculateDeviationError(observedValue, expectedValue) {
-    return observedValue
-      .sub(expectedValue)
+    return this.normalizePriceFeedDecimals(observedValue)
+      .sub(this.normalizePriceFeedDecimals(expectedValue))
       .mul(this.toBN(this.toWei("1"))) // Scale the numerator before division
```

Comment:
Should the 1 not be normalized to the correct precision?

Reply:
nope. deviation error is scaled to 1e18. I refer to this a bit in the comment. As a result, we normalize all numbers to 1e18 and then keep `this.toBN(this.toWei("1"))` scaled accordingly. This returns the 1e18 scalled number for the deviation error.